model/src/pyrenew/process/rtperiodicdiff.py

```diff
@@ -188,7 +192,9 @@ def sample(
         )[0]
 
         return RtPeriodicDiffProcessSample(
-            rt=self.broadcaster(jnp.exp(log_rt.flatten()), duration),
+            rt=SampledValue(
+                self.broadcaster(jnp.exp(log_rt.value.flatten()), duration)
+            ),
```

Comment:
As above, I think we should probably pass `self.t_start` and `self.t_unit` here.

Reply:
Noted